ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -171,6 +171,83 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: azure-ipi-baselinecaps-v412-f28
```

Comment:
I think you can remove this job now, because it was changed to `azure-ipi-arm-mixarch-f14` in https://github.com/openshift/release/pull/43676